house/views.py

```diff
@@ -13,11 +16,42 @@ def global_page(request):
 
 
 def house_login(request):
-    raise NotImplementedError
+
```

Comment:
Please remove unnecessary empty lines and unnecessary comments

Reply:
Also please add tests for the form login code

Reply:
> Also please add tests for the form login code

I am working on that

Reply:
removed unecessary empty lines and unnecessary comments

Reply:
I think there are still too many comments.
“Good code is self documenting"